src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -116,13 +122,13 @@ public void deleteTag(Tag tag) throws PersonNotFoundException, DuplicatePersonEx
     public void addSchedule(Schedule schedule) {
         addressBook.addSchedule(schedule);
         addressBook.sortSchedules();
-        indicateAddressBookChanged();
+        indicateScheduleListChanged();
```

Comment:
I think you should still keep the indicateAddressBookChangedEvent as it is used to save the AddressBook data